content/plugin/log/filtering.mdx

```diff
@@ -0,0 +1,80 @@
+---
+page_title: 'Home - Plugin Development: Filtering Log Output'
+description: >-
+  Filter your provider's log outputs by subsystem and choose the log verbosity level.
+---
+
+# Filtering Log Output
+
+Terraform, providers, and provider SDKs can all write logs that provide insight into operations and help you diagnose bugs. This page explains how to turn on logging, filter log output, and manage log format.
+
+## Logging Environment Variables
+
+Logging is off for all subsystems by default. Set one or more of the the logging environment variables below to either [`JSON` format](#log-format) or a [verbosity level](#logging-levels) to write logs for a Terraform run.
+
+- To write all logs from Terraform binary, all providers, and all provider SDKs, set `TF_LOG`.
+
+   -> **Note**: We recommend setting `TF_LOG=TRACE` when you report bugs to issue trackers. 
+
+- To write only a subset of logs (e.g., for a single provider), do not set `TF_LOG` and instead set specific logging environment variables for your use case.
+
+Some environment variables take precedence over others. For example, if you set `TF_LOG=TRACE` and `TF_LOG_PROVIDER_AZUREM=WARN`, all provider logs will be written at the `TRACE` level because `TF_LOG` takes precedence.
+
+| Variable | Logs Written By | Precedence and Usage |
+| --------------|-------------|-------------|
+| `TF_LOG` | All loggers (Terraform, SDKs, providers) | Overrides all other logging environment variables. |
+| `TF_LOG_CORE` | Terraform binary | Does not include providers. |
+| `TF_LOG_PROVIDER` | All providers and provider SDKs used during the run | Overrides logging environment variables for specific providers (e.g., `TF_LOG_PROVIDER_AWS`) and provider SDKs (e.g., `TF_LOG_PROTO`). Only affects logs from Terraform CLI runs and does not affect logs from provider [acceptance tests](#testing). |
```

Comment:
Okay @bflad I tried to clarify that caveat that we talked about where this only works during CLI runs and not during acceptance testing. I also added more context to the testing section to try to explain a bit more/remind folks of this again.  Thoughts? We can edit more if need be!